### DISCUSSION TITLE
GUI for configuring hyprland

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#8125] [Show and tell]
SO, I have made this recently: https://github.com/nnyyxxxx/hyprgui it is a GUI for configuring Hyprland built entirely in Rust 🦀 I have made an issue before regarding this BUT that was before I decided to remake it.

The point of this discussion to gather opinions on it from most notably @vaxerski and other people apart of HyprWM, as I have spent a great deal of time on it. I have linked a Preview below of what it currently looks like.

It's populated with most of the values from the documentation, includes tooltips explaining each feature (Uses descriptions provided in documentation), etc.

The GUI also includes it's own custom-made parser: https://github.com/nnyyxxxx/hyprparser

I know there is another GUI that exists (Hyprset, was informed by a member of HyprWM) but it seems outdated and incomplete.

The GUI uses gtk, btw.

![image](https://github.com/user-attachments/assets/2e1607b1-88df-4edf-94d8-27c1d5c6af3c)

**Comment by vaxerski** [comment]
I still don't know what you want me to say

**Reply by ghost** [reply]
as said in the post i would like your opinions on it, try it out, see how you like it !! ive put a great deal of time into it + its only a couple days old (first implementation was in py), ive seen in the server that you were thinking about making a settings app for configuring Hyprland as well (are you still planning on doing that??). The GUI still needs more work put into it though, but as of right now it's pretty feature complete.

**Reply by vaxerski** [reply]
> are you still planning on doing that??

yes, in the long term.
